docs/contributor_guide/contributing_code.rst

```diff
@@ -23,8 +23,8 @@ Reductions
 Reduction constructors require a parameter corresponding to an estimator that
 implements the :code:`fit` method with the :code:`sample_weight` argument.
 Parity constraints for reductions are expressed via instances of various
-subclasses of the class :code:`fairlearn.reductions.Moment`. Formally,
-instances of the class :code:`Moment` implement vector-valued random variables
+subclasses of the class :class:`fairlearn.reductions.Moment`. Formally,
+instances of the class :class:`.Moment` implement vector-valued random variables
```

Comment:
```suggestion
instances of the class :class:`Moment` implement vector-valued random variables
```

You shouldn't need that `.`, why is it needed?

Reply:
Oddly enough sometimes this does change whether it renders as "code" (not clickable) or "code + link" (clickable). However, the behavior is not consistent in that sometimes even leaving the `.` out works just fine.

Reply:
not clickable without `.` 
![image](https://user-images.githubusercontent.com/10245648/103038713-fd410200-4523-11eb-9970-a36ac9105a10.png)
clickable with `.`:
![image](https://user-images.githubusercontent.com/10245648/103038746-134ec280-4524-11eb-86a6-889a60330b5d.png)

Reply:
I've seen things like that before. It's something to do with what Python things are 'in scope' at the time the line is read in by Sphinx.

Reply:
yes, whenever they're not a hyperlink, it means sphinx can't find them. You can refer to them with a full path e.g. `fairlearn.module.class` which would render as `fairlearn.module.class` and if you want only the class name, you can have it as `~fairlearn.module.class`. You can also set the scope in a document with the `.. currentmodule::` directive.

Reply:
So you're saying `~fairlearn.module.Class` is preferred over `.Class`? Because in terms of the rendered result there's no difference (both look the same and both have the link)